src/modules/pm.c

```diff
@@ -108,6 +110,30 @@ static int on_new_suspend(sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bu
 
 /* Listener on logind session.Active for current session */
```

Comment:
Can you update the comment here explaining what are we watching now?

Reply:
Yes I'll update this once I fully understand how you want to approach this.